substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ClassEntry.java

```diff
@@ -313,8 +364,99 @@ public int lowpc() {
      *
      * @return the highest code section offset for compiled method code belonging to this class
      */
+    @SuppressWarnings("unused")
```

Comment:
Not sure a public method really needs this.

Reply:
Yes, removed.